DatadogRUM/Sources/Integrations/TelemetryReceiver.swift

```diff
@@ -336,6 +336,8 @@ private extension TelemetryUsageEvent.Telemetry.Usage {
             self = .telemetryCommonFeaturesUsage(value: .setGlobalContext(value: .init()))
         case .setUser:
             self = .telemetryCommonFeaturesUsage(value: .setUser(value: .init()))
+        case .setAccount:
+            self = .telemetryCommonFeaturesUsage(value: .setAccount(value: .init()))
```

Comment:
question/ This seems not used; do we have backlog item to actually send it?

Reply:
No blockers. I forgot about it to be honest.
We need to do it for setUser too.

Reply:
👍 We can do it separately, let's just track it.